deploy/charts/policy-approver/templates/role.yaml

```
@@ -12,3 +12,7 @@ rules:
   resources: ["leases"]
   verbs: ["get", "update"]
   resourceNames: ["policy.cert-manager.io"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "list", "watch", "update"]
+  resourceNames: ['{{ include "cert-manager-policy-approver.name" . }}-tls']
```

Suggestion: might make sense to rename to match the new project name 
```suggestion
  resourceNames: ['{{ include "cert-manager-approver-policy.name" . }}-tls']
```